examples/website/plot/plot-layer/axes-layer.ts

```diff
@@ -222,29 +236,29 @@ export default class AxesLayer extends Layer {
       0, -1, 0, 0, -1, 0
     ];
 
-    const grids = new Model(gl, {
+    const grids = new Model(gl.device, {
       id: `${this.props.id}-grids`,
       vs: gridVertex,
       fs: fragmentShader,
+      bufferLayout: this.getAttributeManager().getBufferLayouts(),
```

Comment:
Note: When `bufferLayout` is omitted, type checking still passes and the layer silently displays nothing. To make updates to v9 a bit easier, I wonder if the types could be more strict about this?

Reply:
luma will [autocreate the buffer layout](https://github.com/visgl/luma.gl/blob/master/modules/core/src/adapter/resources/render-pipeline.ts#L31-L34) for any attributes passed (so here just what is defined in the `geometry`, so I can see the logic in `bufferLayout` being optional. However, this pattern of adding more attributes via `bufferLayout` seems to be very common (at least in deck) and so maybe it would be good to make the type required, so that the user has to explicitly pass in `[]` to signal no additional buffers. @ibgreen thoughts?

In terms of failing silently, luma does complain if it cannot match a buffer, e.g.:

```
Model(PlotLayer-surface-surface): Ignoring buffer "instancePickingColors" for unknown attribute "instancePickingColors"
```